jni/tests/faiss_wrapper_test.cpp

```diff
@@ -282,6 +287,7 @@ TEST(FaissQueryIndexTest, BasicAssertions) {
             delete it;
         }
     }
+    std::cout << "Test end";
```

Comment:
Do we need this?

Reply:
not really. Seems to be a remnant of debugging